src/ResourceManager/Sql/Commands.Sql/Failover Group/Services/AzureSqlFailoverGroupAdapter.cs

```diff
@@ -107,11 +107,11 @@ internal AzureSqlFailoverGroupModel UpsertFailoverGroup(AzureSqlFailoverGroupMod
             partnerServers.Add(partnerServer);
 
             ReadOnlyEndpoint readOnlyEndpoint = new ReadOnlyEndpoint();
-            readOnlyEndpoint.FailoverPolicy = model.ReadOnlyFailoverPolicy;
+            readOnlyEndpoint.FailoverPolicy = model.ReadOnlyFailoverPolicy == null ? AllowReadOnlyFailoverToPrimary.Disabled.ToString() : model.ReadOnlyFailoverPolicy;
```

Comment:
This change and the other 3 symmetric do not actually influence behavior and are not needed